cfgov/v1/tests/test_filterable_list_form.py

```diff
@@ -89,6 +89,12 @@ def test_filter_by_title(self):
         self.assertEquals(len(page_set), 1)
         self.assertEquals(page_set[0].specific, page1)
 
+    def test_validate_date_after_1900(self):
```

Comment:
For someone coming to this code in the future, it might be nice to have another test which is identical to this one but works because the date is valid. That way it'd be easier to see that the difference in expected behavior is due only to the date.

Reply:
Makes sense. Added that test in 54de85a8111b458651806f8b840c026c1a816bd4.